reactive/kubernetes_master.py

```diff
@@ -2363,7 +2439,7 @@ def poke_network_unavailable():
 
     for node in nodes:
         node_name = node['metadata']['name']
-        url = 'http://localhost:8080/api/v1/nodes/{}/status'.format(node_name)
+        url = '{}/api/v1/nodes/{}/status'.format(public_server, node_name)
         with urlopen(url) as response:
```

Comment:
This function will need some more work. The requests are going to fail - they do not include a token. I would use the admin token.